packages/terra-form-checkbox/src/Checkbox.scss

```diff
@@ -46,6 +46,12 @@
       max-width: 100%; //stops content overflow on IE 10/11
       user-select: none;
     }
+
+    .label-text.is-mobile {
+      display: inline-block;
+      vertical-align: var(--terra-form-checkbox-is-mobile-label-text-vertical-align, 0);
```

Comment:
You can remove the `, 0` part. e.g. `vertical-align: var(--terra-form-checkbox-is-mobile-label-text-vertical-align);`